modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -112,6 +116,55 @@ This requires an Enterprise license.
 This mode is not recommended for production clusters. Only set to `off` if you need to move partitions manually.
 |===
 
+== Intra-broker partition balancing
+
+[CAUTION]
+====
+This is a beta feature for v24.2 and is not recommended for use for production clusters.
+
+The feature is disabled by default. To unlock this feature, run this command with a superuser:
```

Comment:
I'd move this line up to have 1 paragraph (instead of 2) in the Caution admonition. And I'd probably make it a Note instead of a Caution, unless PM requested otherwise.